docker-hub/publish/insights-analytics.md

```diff
@@ -0,0 +1,76 @@
+
+There are automated systems that frequently check for new versions of your images, and we have tried to distinguish between those 'version checks' and real user downloads by inspecting the order and timing of API calls from the same IP address. This will provide more insight into customer behavior. You can inspect the rules we have applied in the Action Classification Rule Set section of this document. We welcome [feedback](https://forms.gle/nb7beTUQz9wzXy1b6){: target="_blank" rel="noopener" class="_"} and questions on these rules.
+
+## Raw data
+
+In your raw data CSV you will have access to the data points listed below. You can request raw data for a complete week (Monday through Sunday) or for a complete month (available on the first day of the following month). **Note:** each action is represented as a single row.
+
+  * Type (industry)
+  * Host (cloud provider)
+  * Country (geolocation)
+  * Timestamp
+  * Namespace
+  * Repo
+  * Reference (digest is always included, tag is provided when available)
+  * HTTP method
+  * Action, one of the following:
+    * Pull by tag
+    * Pull by digest
+    * Version check
+  * User-Agent Tool
+
+## Summary data
+
+In your summary data CSV, you will have access to the data points listed below. You can request summary data for a complete week (Monday through Sunday) or for a complete month (available on the first day of the following month).
+
+For every namespace, repository, and reference (tag or digest):
+
+  * Unique IP address count
+  * Pulls by tag count
+  * Pulls by digest count
+  * Version check count
+
+## Data definitions
+
+| Data point | Definition |
+|:-----|:--------|
+| Action | An Action represents the multiple request events associated with a `docker pull`. We have applied rules to these events so that the data is more meaningful in analyzing user behavior and intent. An Action can be filtered into three distinct categories: version check, pull by tag, and pull by digest. Each Action is represented as a single row in the raw data. |
+| Version check  |  This is a filter on the Action data point. It is a speculation of user intent. Includes: HEAD by tag not followed by a GET (from the same IP address within a 5-second window). Excludes: HEAD by digest |
+| Pull by tag | This is a filter on the Action data point. It is a speculation of user intent. Includes: GET (by digest or by tag). If the GET is immediately preceded by a HEAD by tag (from the same IP address within a 5-second window), then the GET and HEAD together are counted as a single Pull by Tag. If the GET by tag is immediately followed by another GET (from the same IP address within a 5-second window, but a different digest), then the two GETs are counted as a single Pull by Tag. |
```

Comment:
The data here feels a little duplicated with the `Action classification ruleset` below... Not a big deal though.